docs/api/field_type_reference.md

```diff
@@ -1501,6 +1501,157 @@ new FieldType\Value([
 ]);
 ```
 
+### GraphQL Field Type operations
+
+To get a Matrix Field Type, you will need to specify a Content ID, a Content Type, and a Field Type. 
+The types used for input and returned are named after the Type and the Field: 
+
+- `{TypeIdentifier}{FieldIdentifier}RowInput`
```

Comment:
We should add at least two examples of this. Suggestion:
- `dish.nutritionFacts`
- still thinking, maybe I'll think of one later :)